packages/instantsearch-react/src/utils.js

```diff
@@ -12,3 +12,20 @@ export function isSpecialClick(event) {
 export function capitalize(key) {
   return key.length === 0 ? '' : `${key[0].toUpperCase()}${key.slice(1)}`;
 }
+
+export function assertFacetDefined(searchParameters, searchResults, facet) {
+  const wasRequested =
+    searchParameters.isConjunctiveFacet(facet) ||
+    searchParameters.isDisjunctiveFacet(facet);
```

Comment:
Well the name of the facet used when defining a hierarchical facet is the one used throughout the code. We usually don't use the real facet name. And it seems that's what you would be checking here.